package.json

```diff
@@ -28,6 +28,7 @@
     "@types/express": "^4.0.35",
     "@types/express-session": "^0.0.32",
     "@types/jest": "^18.1.1",
+    "@types/node-fetch": "^1.6.7",
```

Comment:
It looks like this is pulling in typings for `node-fetch` 1.6.7, whereas we appear to be installing `node-fetch` 1.6.3 below.